src/Views/ShortcutsView.vala

```diff
@@ -124,4 +98,17 @@ public class ShortcutOverlay.ShortcutsView : Gtk.Grid {
         add (new Gtk.Separator (Gtk.Orientation.VERTICAL));
         add (column_end);
     }
+
+    private void add_shortcut_entries (Gee.ArrayList<ShortcutEntry> entries, Gtk.Grid column) {
+        for (int i = 0; i < entries.size; i++) {
+            var label = new ShortcutLabel (entries[i]);
+
+            if (i + 1 == entries.size) {
+                label.get_style_context ().add_class ("last-entry");
```

Comment:
Adding per-entry CSS should definitely not be needed. If we want to do something about the last item in a column, for example, that can always be done with a `:last-child` selector. In that case, I'd recommend switching back to a foreach for simplicity.